sre-recipes/recipes/recipe_1/currencies_recipe.py

```diff
@@ -0,0 +1,138 @@
+
+# -*- coding: utf-8 -*-
+"""
+This module contains the implementation of recipe 1
+"""
+
+import logging
+import subprocess
+from recipe import Recipe
+
+
+class CurrenciesRecipe(Recipe):
+    """
+    This class implements recipe 1, which purposefully
+    introduces latency into the frontend service.
+    """
+
+    @staticmethod
+    def _run_command(command):
+        """Runs the given command and returns any output and error"""
+        process = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
+        output, error = process.communicate()
+        return output, error
+
+    @staticmethod
+    def _deploy_state(state):
+        """
+        Sets an environment variable CONVERT_CURRENCIES to given state
+        and updates the state accordingly
+        """
+        state_str = str(state).lower()
+        set_env_command = f"kubectl set env deployment/frontend CONVERT_CURRENCIES={state_str}"
+        get_pod_command = """kubectl get pod -l app=frontend -o \
+            jsonpath=\"{.items[0].metadata.name}\""""
+        logging.info('Setting env variable: %s', set_env_command)
+        logging.info('Getting pod: %s', get_pod_command)
+
+        CurrenciesRecipe._run_command(set_env_command)
+        service, error = CurrenciesRecipe._run_command(get_pod_command)
+        service = service.decode("utf-8").replace('"', '')
+        delete_pod_command = f"kubectl delete pod {service}"
+        logging.info('Deleting pod: %s', delete_pod_command)
+        CurrenciesRecipe._run_command(delete_pod_command)
+
+    def break_service(self):
+        """
+        Rolls back the working version of the given service and deploys the
+        broken version of the given service
+        """
+        print("Deploying broken service...")
+        self._deploy_state(True)
+        logging.info('Deployed broken service')
+
+    def restore_service(self):
+        """
+        Rolls back the broken version of the given service and deploys the
+        working version of the given service
+        """
+        print("Deploying working service...")
+        self._deploy_state(False)
+        logging.info('Deployed working service')
+
+    @staticmethod
+    def _get_user_input():
+        """Prompts users for an answer"""
+        answer = input('Your answer: ')
+        return answer
+
+    def _service_multiple_choice(self):
+        """
+        Displays a multiple choice quiz to the user about which service
+        broke and prompts the user for an answer
+        """
+        print('Which service broke?')
+        print('\t[a] ad service')
+        print('\t[b] cart service')
+        print('\t[c] checkout service')
+        print('\t[d] currency service')
+        print('\t[e] email service')
+        print('\t[f] frontend service')
+        print('\t[g] payment service')
+        print('\t[h] product catalog service')
+        print('\t[i] recommendation service')
+        print('\t[j] redis')
+        print('\t[k] shipping service')
```

Comment:
nit: This would probably look better as printing a [multiline string](https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string)

Reply:
fixed!